code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -381,3 +381,22 @@
 	R.module.add_module(S, FALSE, TRUE)
 
 	return 1
+
+/obj/item/borg/upgrade/ai
+	name = "b.o.r.i.s. module"

```

Comment:
I think you can call it `B.O.R.I.S.`, lower case isn't used on acronyms. Could also just go BORIS.